module2/presentation_virtual.md

```diff
@@ -175,39 +137,32 @@ ___
 
 ## Zadanie 1
 
-Przekształć klasę `Cargo` w interfejs z 4 czysto wirtualnymi metodami.
-
-```cpp
-virtual size_t getPrice() const = 0;
-virtual std::string getName() const = 0;
-virtual size_t getAmount() const = 0;
-virtual size_t getBasePrice() const = 0;
-```
-
-___
-
-## Zadanie 1 cd.
-
-Utwórz 3 pochodne klasy `Cargo`:
+Zadanie 1:
+Przekształć klasę `Cargo` w interfejs z 4 wirtualnymi metodami. Utwórz 3 pochodne klasy:
 
 * `Fruit`
 * `Alcohol`
 * `Item`
 
 Klasa `Fruit` powinna mieć dodatkową zmienną określającą czas do zepsucia oraz `operator--` który będzie odejmował ten czas o 1.
-Metoda `getPrice()` powinna redukować cenę odpowiednio wraz z czasem psucia naszego owocu.
+Metoda `getPrice()` powinna być redukowana odpowiednio wraz z czasem psucia naszego owocu.
 
 Klasa `Alcohol` powinna mieć dodatkową zmienną określającą procentowy udział spirytusu.
 Metoda `getPrice()` powinna być proporcjonalnie wyższa w zależności od mocy alkoholu.
-Należy ustalić bazową cenę za spirytus 96%.
+Należy ustalić bazową cenę za spirytus(96%).
 
 Klasa `Item` powinna mieć dodatkową zmienną enum określającą rzadkość przedmiotu (common, rare, epic, legendary).
 Metoda `getPrice()` powinna być adekwatnie wyliczana od poziomu rzadkości przedmiotu.
 
+```cpp
+virtual size_t getPrice() const = 0;
+virtual std::string getName() const = 0;
+virtual size_t getAmount() const = 0;
+virtual size_t getBasePrice() const = 0;
+```
+
 ___
 
 ## Zadanie 2
 
-Wykorzystując wspólną klasę bazową `Cargo` spróbuj przechowywać wszystkie towary w jednym wektorze w klasie `Ship`.
-
-Dodaj funkcję `void load(std::shared_ptr<Cargo> cargo)`, która dodaje towar i (dla chętnych) `void unload(Cargo* cargo)`, która usuwa towar z obiektu klasy `Ship`.
+Wykorzystując wspólną klasę bazową `Cargo` spróbuj przechowywać wszystkie towary w jednym wektorze w klasie `Ship`. Dodaj funkcję `void load(std::shared_ptr<Cargo> cargo)`, która dodaje towar i (dla chętnych) `void unload(Cargo* cargo)`, która usuwa towar z obiektu klasy `Ship`.
```

Comment:
Why have you pushed those all files?

Reply:
My bad.  I've messed up with previous merge, then tried to fix it ... and I've made situation even worse.